spec/fixtures/state_file/fed_return_xmls/2023/md/zeus_two_w2s.xml

```diff
@@ -232,7 +232,7 @@
       <EmployerEIN>001245768</EmployerEIN>
       <EmployerNameControlTxt>BTB</EmployerNameControlTxt>
       <EmployerName>
-        <BusinessNameLine1Txt>Brendan T. Byrne State Forest</BusinessNameLine1Txt>
+        <BusinessNameLine1Txt>Brendan T Byrne State Forest</BusinessNameLine1Txt>
       </EmployerName>
```

Comment:
this was erroring out in the state return. I'm assuming federal xmls will not have errors with (.)s so wanted fix this